Makefile

```diff
@@ -1,61 +1,5 @@
-GO	:= $(shell which go)
-GLIDE	:= $(shell which glide)
-APP	?= github-authorized-keys
-INSTALL_DIR ?= /usr/local/sbin
-
 include Makefile.*
```

Comment:
add `-include build-harness/golang/Makefile*`

Make `build-harness` a submodule pinned to `master@HEAD` or add a target that runs `git clone`